libs/UIHelpers/transfer_function_widget.cpp

```diff
@@ -517,21 +555,19 @@ void TransferFunctionWidget::set_Quantiles(float min, float max)
   m_quantiles[1] = max;
 }
 
-
-void TransferFunctionWidget::get_Quantiles(float& min, float& max)
+void TransferFunctionWidget::get_Quantiles(float &min, float &max)
 {
   min = m_quantiles[0];
   max = m_quantiles[1];
 }
 
-void TransferFunctionWidget::load_embedded_preset(const uint8_t* buf,
-  size_t size,
-  const std::string& name)
+void TransferFunctionWidget::load_embedded_preset(const uint8_t *buf,
+                                                  size_t size,
+                                                  const std::string &name)
```

Comment:
Same thing with the `)` here. Could be another quirk of the formatter.

Reply:
reformatted, nothing much changed,